packages/google-cloud-translate/package.json

```diff
@@ -4,21 +4,33 @@
   "version": "7.2.0",
   "license": "Apache-2.0",
   "author": "Google Inc.",
+  "type": "module",
   "engines": {
-    "node": ">=12.0.0"
+    "node": ">=14.18.0"
```

Comment:
Was there an issue with 14.0.0?

Reply:
good call, just using it while I was testing.